N/npth/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "npth"
```

Comment:
The upstream library is apparently typed
```suggestion
name = "nPth"
```
If you want to change this name, you also need to rename the directory accordingly